mix.exs

```diff
@@ -75,6 +75,7 @@ defmodule Archethic.MixProject do
 
       # P2P
       {:ranch, "~> 2.1", override: true},
+      {:mmdb2_decoder, "~> 3.0"},
```

Comment:
Is it better than maxmind geo database?

Reply:
This is the same database, but just use a lighter library to fetch the data.
Geolix and GeolixAdapterMmdb2 were to eavy and finaly use Mmdb2Decoder